web/js/app.js

```diff
@@ -709,5 +709,48 @@ LR.utils.removeCollectionEntry = async function(event, itemID, itemElem, collect
         //This should suffice for now.
         console.error("There was an error gathering information to remove an item from the collection")
         console.log(itemElem)
-    })
-}
\ No newline at end of file
+    })  
+}
+
+/**
+ * Check if the user from session is the creator of some given Linked Data node or URI.
+ * 
+ * @param {string} user The agent ID of the user in session
+ * @param {object || string} item A Linked Data node or URI
+ * @return {Boolean}
+ */
+LR.utils.isCreator = async function(agentID, item){
+    let creator
+    let creatorID
+    if(typeof item === "string"){
+        //It is probably just a URL.  We need to fetch the object and perhaps expand it
+        item = await fetch(item).then(response => response.json()).catch(err => {
+            console.error(err)
+            return {}
+        })
+    }
+    
+    // Now item is an object and we expect it to have creator on it in some fashion.
+    if(item.creator){
+        creator = item.creator
+    }
+    else if(item.body && item.body.creator){
```

Comment:
Yes, since Lived Religion uses `creator` as a primitive, it should be there.  If not, it is bad data, and we can't compare.  thought a little too high level here.